_episodes/55.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: episode
+title: "55: Commander Bun Bun peaks at Peaka"
+date: 2024-01-18
+tags: trino peaka
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "Episode 55: Commander Bun Bun peaks at Peaka"
+  time: 0
+- title: "Trino releases"
+  desc: "437-438"
+  time:
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [Mustafa Sakalsiz](https://linkedin.com/in/sakalsiz), CEO at
+  [Peaka](https://www.peaka.com/)
+* [Ali Tekin](https://www.linkedin.com/in/alitekin/), Principal Software
+  Architect at [Peaka](https://www.peaka.com/)
+
+## Releases 437-438
+
+[Trino 437](https://trino.io/docs/current/release/release-437.html)
+
+* Support for configuring compression codecs
+* Support for `char` values in the `to_utf8()` and `lpad()` functions
+* Improved performance for Delta Lake queries without table statistics
+* Improved performance for Iceberg queries with filters on `ROW` columns
+
+[Trino 438](https://trino.io/docs/current/release/release-438.html)
+
+* Support for access control with Open Policy Agent
+* Support for `ALTER COLUMN ... DROP NOT NULL` in Iceberg and PostgreSQL
+* Support for configuring page sizes in Delta Lake, Hive, and Iceberg
+* Better type support for the `reduce_agg()` function
+
```

Comment:
Additional note here to https://trino.io/blog/2024/02/06/opa-arrived

Reply:
Also lets add Trino Gateway 5 - https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-5-24-jan-2024

Reply:
and charts 016, 0.17 and 0.18 - separate releases but just a few minor improvments